src/Html/Builder.php

```diff
@@ -161,7 +161,7 @@ public function parameterize($attributes = [])
      */
     protected function isCallbackFunction($value, $key)
     {
-        return Str::startsWith(trim($value), ['$', '$.', 'function']) || Str::contains($key, 'editor');
+        return Str::startsWith(trim($value), config('datatables-html.callback', ['$', '$.', 'function'])) || Str::contains($key, 'editor');
```

Comment:
Maybe we can use the config property instead since we already injected it?

```
$this->config->get('datatables-html.callback', ['$', '$.', 'function'])
```